app/models/supported_ssg.rb

```diff
@@ -68,19 +68,25 @@ def revision
 
     # Multilevel map of latest supported SSG for OS major and minor version
     def latest_map
-      clear if ssg_ds_changed?
-
-      @latest_map ||= build_latest_map
+      Rails.cache.fetch("SupportedSsg/datastreams/#{Revision.datastreams}/map") do
+        build_latest_map
+      end
     end
 
     def by_os_major
       all.group_by(&:os_major_version)
     end
 
     def by_ssg_version
-      clear if ssg_ds_changed?
+      Rails.cache.fetch("SupportedSsg/datastreams/#{Revision.datastreams}/by-ssg") do
+        all.group_by(&:version)
+      end
+    end
 
-      @by_ssg_version ||= all.group_by(&:version)
+    def clear(to_clear = revision)
```

Comment:
Can you please elaborate on the default value of `to_clear`? It is being set once the code is loaded by calling `revision`.

Reply:
Never mind. It is real-time evaluated.